From 8b96aac1c127487f4b6bdf65f3486eda2a5af8d0 Mon Sep 17 00:00:00 2001
From: Martynas Pumputis <m@lambda.lt>
Date: Mon, 1 Jul 2019 16:45:47 +0200
Subject: [PATCH] overlayfs: Disable l is in-use by u/w check

Just to work around until Docker has fixed the underlying issue.

Signed-off-by: Martynas Pumputis <m@lambda.lt>
---
 fs/overlayfs/super.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/fs/overlayfs/super.c b/fs/overlayfs/super.c
index b368e2e102fa..7bba0e92d7ad 100644
--- a/fs/overlayfs/super.c
+++ b/fs/overlayfs/super.c
@@ -1314,10 +1314,10 @@ static int ovl_get_lower_layers(struct super_block *sb, struct ovl_fs *ofs,
 			goto out;
 
 		err = -EBUSY;
-		if (ovl_is_inuse(stack[i].dentry)) {
-			pr_err("overlayfs: lowerdir is in-use as upperdir/workdir\n");
-			goto out;
-		}
+		//if (ovl_is_inuse(stack[i].dentry)) {
+		//	pr_err("overlayfs: lowerdir is in-use as upperdir/workdir\n");
+		//	goto out;
+		//}
 
 		err = ovl_setup_trap(sb, stack[i].dentry, &trap, "lowerdir");
 		if (err)
-- 
2.21.0

